src/helpers.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Wouldn't these conflict with functions that might be defined by the underlying project already?
Especially as they are necessary for the functioning of the library.

Wouldn't it be better to have a dedicated `Helpers` class here?

Reply:
We can get rid of these helper functions, or wrap inside our namespace. I think its better than an abstract class with static functions

Reply:
> or wrap inside our namespace

You mean as functions directly, instead of static methods?

Reply:
Yes, like \Telegram\app().

Those helpers are for internal use, so in most cases, an user will not use those helpers